bbpower/foreground_loading.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+
+from fgbuster.component_model import CMB, Dust, Synchrotron, AnalyticComponent, FreeFree 
+
+hplanck = 6.626068e-34
+kboltz = 1.3806503e-23
+
+class FGModel:
+    def __init__(self, config):
+        self.load_foregrounds(config)
+        return 
+
+    def load_foregrounds(self, config):
+        """
+        Loads the foreground models and prepares the unit conversions to K_CMB units. 
+        """
+        self.components = {}
+        for key in config['fg_model']:
+            self.components[key] = {}
+
+        for key, component in config['fg_model'].items(): 
+            nu0 = component['parameters']['nu0']
+            sed_fnc = get_fgbuster_sed(component['sed'])
+            self.components[key]['sed'] = sed_fnc(**component['parameters'], units='K_RJ')
+            self.components[key]['cmb_n0_norm'] = CMB('K_RJ').eval(nu0) * nu0**2
+            self.components[key]['nu0'] = nu0
+            self.components[key]['spectrum_params'] = component['spectrum']
+        return 
+    
+
+class FGParameters: 
+    def __init__(self, config):
+        self.define_parameters(config)
+        return 
+
+    def define_parameters(self, config):
```

Comment:
It'd be good to document just a bit exactly what ordering you're using, and what each of the different attributes here are